controllers/storagecluster/reconcile.go

```diff
@@ -53,7 +53,7 @@ const (
 	ReconcileStrategyIgnore ReconcileStrategy = "ignore"
 	// ReconcileStrategyManage means always reconcile
 	ReconcileStrategyManage ReconcileStrategy = "manage"
-	// ReconcileStrategyStandalone also means never reconcile (NooBaa)
+	// ReconcileStrategyStandalone means to renconcile the component exclusively (NooBaa)
```

Comment:
Nit: typo reconcile

Reply:
The typo needs to be fixed. but could be fixed in a follow-up PR too.

I don't fully understand where the statement of standalone-noobaa vs external mode comes from. If I read the code correctly, in the noobaa-standalone mode, the noobaa reconciler just does not need to worry about the cephcluster at all any more (external or internal).

Reply:
Good catch